docs/Setting_3rd_party_cert_environment.md

```diff
@@ -0,0 +1,76 @@
+# Configuring environment with a third party certificate
+
+## Disclaimer
+The purpose of this document is to help a developer during the development and testing 
+process where they require a trusted, third party certificate. The process detailed 
+below should be used for **TESTING PURPOSES** only and by no means intends to provide 
+instructions for configuring an production environment.
+## Motivation
+
+Every certificate has to be signed by some CA (Certification Authority). There are many
+CA's out there. Some are big, like Google or Amazon, and some are small CA's.
+One can also create it own private CA and signed it own certificates.
+Many Companies (mostly big companies) choose to create their own CA and issue 
+their own certificates. This doc purpose is to be a walk-through on how to issue 
+your own certificate and to configure your env to work with your CA and issued certificate.
+
+## General information
+- Python comes with a list of trusted CAs. Every certificate signed by those CAs 
+will be considered as trusted certificate. Those CAs are located in a file called cacerts.pem. 
+  To get the path of this file, run in `python3 -c "import certifi; print(certifi.where())"`  
+
+## Walk-through
+### Creating your own CA
+In this section we are creating our own CA. we will later use this CA to issue certificate for Conjur
+and will add this CA to the trusted CAs list so that the CLI will trust the certificate 
```

Comment:
```suggestion
. We will add this CA to the trusted CAs list so that the CLI will trust the certificate 
```

Reply:
IMO it is more than just trusting. It is being able to trust it verify the certificate against its own system-level list of certs